src/scenarios/personalization/ProgressGet/ProgressGet.tsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from 'react';
+import {
+  Button,
+  Container,
+  Divider,
+  Form,
+  Header,
+  Label,
+  Segment,
+} from 'semantic-ui-react';
+import {
+  useScenarioHost,
+  VariableSearch,
+} from '@axinom/mosaic-fe-samples-host';
+import { getApolloClient } from '../../../apollo-client';
+import { getProgressQuery } from './graphql-documents';
+
+export const ProgressGet: React.FC = () => {
+  const { activeProfile, logger } = useScenarioHost();
+  const [userAccessToken, setUserAccessToken] = useState<string>();
+  const [key, setKey] = useState<string>();
+
+  const apolloClient = getApolloClient(
+    new URL('graphql', activeProfile.personalizationServiceBaseURL).href,
+  );
+
+  const getProgressValue = async (): Promise<void> => {
+    try {
+      const result = await apolloClient.query({
+        query: getProgressQuery,
+        variables: {
+          getProgressInput: {
+            keys: [key],
+            scope: 'PROFILE',
+          },
+        },
+        context: {
+          headers: {
+            Authorization: `Bearer ${userAccessToken}`,
+          },
+        },
+        fetchPolicy: 'no-cache',
+      });
+
+      logger.log(`method [${getProgressValue.name}]`, 'output:', result.data);
+
+      if (result.errors) {
```

Comment:
Let's log graphql errors inside the exception handler. The backend API sends a 400 response for errors, therefore this block of code will not execute